iocBoot/iocMuonJaws/config.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
```

Comment:
ioc_desc?

Reply:
The support module should not contain an iocBoot area - I forgot to remove it after generating the directory structure.